patches/@react-navigation+native+6.0.13.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/node_modules/@react-navigation/native/src/useLinking.native.tsx b/node_modules/@react-navigation/native/src/useLinking.native.tsx
```

Comment:
will do a PR for this in react-navigation. https://github.com/react-navigation/react-navigation/pull/10708#issuecomment-1260549184